README.md

```
@@ -25,159 +36,399 @@
+- 모두 읽음을 클릭하면 전체를 확인상태(전체 조회)로 만듬
+
+- 매일 배치를 돌려 1주일이 지난 알림을 자동으로 삭제
+
+</details>
+
+<details>
+<summary>대시보드</summary>
+
+- 매일 스프링 배치로 일/주/월/전체 기간별 순위를 생성
+
+- 도서는 (해당 기간의 리뷰수 * 0.4) + (해당 기간의 평점 평균 * 0.6)으로 계산
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)